### PR TITLE
INFRA: Initialize Acceptance Tests

### LINKED ISSUE #3: INFRA: Initialize Acceptance Test Project

### PR DESCRIPTION
**Pull request by Manasa1191**
closes #3

### CODE REVIEW

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using FluentAssertions;
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs:1]: Add copyright header, remove all empty spaces (lines).

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace InternTrack.Core.Api.Tests.Acceptance.Apis.Home
+{
+    [Collection(nameof(ApiTestCollection))]
+    public partial class HomeApiTests
+    {
+        private readonly InternTrackApiBroker internTrackApiBroker;
+        public HomeApiTests(InternTrackApiBroker internTrackApiBroker) =>
+            this.internTrackApiBroker = internTrackApiBroker;
+
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs:11]: Remove empty line, add space between single liners and multiple liners.

File: InternTrack.Core.Api.Tests.Acceptance/Brokers/ApiTestCollection.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace InternTrack.Core.Api.Tests.Acceptance.Brokers
+{
+    [CollectionDefinition(nameof(ApiTestCollection))]
+    public class ApiTestCollection : ICollectionFixture<InternTrackApiBroker>
+    {
+
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Brokers/ApiTestCollection.cs:6]: Remove empty space

File: InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.Home.cs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿namespace InternTrack.Core.Api.Tests.Acceptance.Brokers
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.Home.cs:1]: Add copyright header.

File: InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.cs [hunk 5]
```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using RESTFulSense.Clients;
+
+
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.cs:4]: Remove empty space and add copyright header

File: InternTrack.Core.Api.Tests.Acceptance/Usings.cs [hunk 6]
```diff
@@ -0,0 +1 @@
+global using Xunit;
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Usings.cs:1]: Remove global using. They are magic code. Not allowed.

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs [hunk 7]
```diff
@@ -0,0 +1,13 @@
+﻿using InternTrack.Core.Api.Tests.Acceptance.Brokers;
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs:1]: Add copyright header

**Comment by glhays** [comment]
You should reference in your Pull Request comment that this request closes a specific issue. Most if not all PR's should reference a open Issue in the repository. This assures that another developer does not commit conflicting code to yours which creates conflicts that will have to be resolved.

Adding the statement below to your PR comment you added in the commit like so.
Closes #(issue number here)

Have questions please ask.

File: InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.cs (hunk 5 above)
Comment (hassanhabib) [reply InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.cs:7]: @Manasa1191 take care of this.

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs [hunk 8]
```diff
@@ -0,0 +1,20 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs:3]: Add line here

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+    public partial class HomeApiTests
+    {
+        [Fact]
+        public async Task ShouldReturnHomeMessageAsync()
+        {
+            string expectedMessage = "Thank you, Mario. But the princess is in another castle!";
+            string actualMessage = await this.internTrackApiBroker.GetHomeMessageAsync();
+            actualMessage.Should().BeEquivalentTo(expectedMessage);
+        }
+
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs:18]: Remove extra line

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs [hunk 10]
```diff
@@ -0,0 +1,19 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+using InternTrack.Core.Api.Tests.Acceptance.Brokers;
+using Xunit;
+
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs:6]: Remove extra line here and add line under copyright.

File: InternTrack.Core.Api.Tests.Acceptance/Brokers/ApiTestCollection.cs [hunk 11]
```diff
@@ -0,0 +1,13 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+using Xunit;
+
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Brokers/ApiTestCollection.cs:5]: Same here extra line and add the line above.

File: InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.Home.cs [hunk 12]
```diff
@@ -0,0 +1,13 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.Home.cs:3]: Add a space.

File: InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.cs [hunk 13]
```diff
@@ -0,0 +1,22 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+using Microsoft.AspNetCore.Mvc.Testing;
+using RESTFulSense.Clients;
+
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.cs:6]: Same here add line above and remove extras

File: InternTrack.Core.Api.Tests.Acceptance/Usings.cs [hunk 14]
```diff
@@ -0,0 +1 @@
+using Xunit;
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Acceptance/Usings.cs:1]: Remove this it is not needed. All usings are placed at the top each file by the coder. Global using are considered magic code and unacceptable in The Standard principles.

**Review by glhays — CHANGES_REQUESTED** [review]
@Manasa1191 new review please address.

File: InternTrack.Core.Api.Tests.Acceptance/Brokers/ApiTestCollection.cs (hunk 11 above)
Comment (Manasa1191) [reply InternTrack.Core.Api.Tests.Acceptance/Brokers/ApiTestCollection.cs:5]: I will look into it . Thank you

File: InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.cs (hunk 13 above)
Comment (Manasa1191) [reply InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.cs:6]: Made following changes.  let me know if any other changes needed.
1. add space after copyright (line 4)
2. remove space between single line statements (line 7)
3. remove space after closing braces (line 19)

Screenshot (41)
@glhays

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+using Xunit;
+
+namespace InternTrack.Core.Api.Tests.Acceptance.Apis.Home
+{
+    public partial class HomeApiTests
+    {
+        [Fact]
+        public async Task ShouldReturnHomeMessageAsync()
+        {
+            string expectedMessage = "Thank you, Mario. But the princess is in another castle!";
```
Comment (hassanhabib) [inline InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs:14]: All tests should be roken into `// given`, `// when` and `// then` just like it shows in the examples, videos and existing code in other projects

Comment (Manasa1191) [reply InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs:14]: Okay . On it 👍🏻

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs [hunk 16]
```diff
@@ -0,0 +1,19 @@
...
+namespace InternTrack.Core.Api.Tests.Acceptance.Apis.Home
+{
+    [Collection(nameof(ApiTestCollection))]
+    public partial class HomeApiTests
+    {
+        private readonly InternTrackApiBroker internTrackApiBroker;
+
+        public HomeApiTests(InternTrackApiBroker internTrackApiBroker) =>
+            this.internTrackApiBroker = internTrackApiBroker;
+
```
Comment (hassanhabib) [inline InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs:17]: There's an extra line here.

File: InternTrack.Core.Api.Tests.Acceptance/Usings.cs (hunk 14 above)
Comment (hassanhabib) [inline InternTrack.Core.Api.Tests.Acceptance/Usings.cs:1]: Why's this file here?

Comment (Manasa1191) [reply InternTrack.Core.Api.Tests.Acceptance/Usings.cs:1]: I am not sure why I have this file . Double checking training videos. Will delete this file if not needed

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs (hunk 16 above)
Comment (Manasa1191) [reply InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs:17]: Hi, Are you saying extra line is there on - line 14?

**Pull request closed without merging**